Explorer/Assets/DCL/Multiplayer/SDK/Systems/WriteAvatarEquippedDataSystem.cs

```diff
@@ -0,0 +1,102 @@
+{
+    [UpdateInGroup(typeof(PresentationSystemGroup))]
+    [UpdateAfter(typeof(PlayerComponentsHandlerSystem))]
+    [LogCategory(ReportCategory.PLAYER_AVATAR_EQUIPPED)]
+    public partial class WriteAvatarEquippedDataSystem : BaseUnityLoopSystem
+    {
+        private readonly IECSToCRDTWriter ecsToCRDTWriter;
+        private readonly IComponentPool<PBAvatarEquippedData> componentPool;
+
+        public WriteAvatarEquippedDataSystem(World world, IECSToCRDTWriter ecsToCRDTWriter, IComponentPool<PBAvatarEquippedData> componentPool) : base(world)
+        {
+            this.ecsToCRDTWriter = ecsToCRDTWriter;
+            this.componentPool = componentPool;
+        }
+
+        protected override void Update(float t)
+        {
+            HandleComponentRemovalQuery(World);
+            UpdateAvatarEquippedDataQuery(World);
+            CreateAvatarEquippedDataQuery(World);
+        }
+
+        [Query]
+        [None(typeof(PBAvatarEquippedData), typeof(DeleteEntityIntention))]
+        private void CreateAvatarEquippedData(in Entity entity, ref PlayerSDKDataComponent playerSDKDataComponent)
+        {
+            PBAvatarEquippedData pbComponent = componentPool.Get();
+            pbComponent.WearableUrns.Clear();
+
+            foreach (URN urn in playerSDKDataComponent.WearableUrns) { pbComponent.WearableUrns.Add(urn); }
+
+            pbComponent.EmoteUrns.Clear();
+
+            foreach (URN urn in playerSDKDataComponent.EmoteUrns) { pbComponent.EmoteUrns.Add(urn); }
+
+            ecsToCRDTWriter.PutMessage<PBAvatarEquippedData, PBAvatarEquippedData>(static (dispatchedPBComponent, pbComponent) =>
+            {
+                dispatchedPBComponent.WearableUrns.Clear();
+
+                foreach (URN urn in pbComponent.WearableUrns) { dispatchedPBComponent.WearableUrns.Add(urn); }
+
+                dispatchedPBComponent.EmoteUrns.Clear();
+
+                foreach (URN urn in pbComponent.EmoteUrns) { dispatchedPBComponent.EmoteUrns.Add(urn); }
+            }, playerSDKDataComponent.CRDTEntity, pbComponent);
+
+            World.Add(entity, pbComponent, playerSDKDataComponent.CRDTEntity);
+        }
+
+        [Query]
+        [None(typeof(DeleteEntityIntention))]
+        private void UpdateAvatarEquippedData(in Entity entity, ref PlayerSDKDataComponent playerSDKDataComponent, ref PBAvatarEquippedData pbComponent)
+        {
+            if (!playerSDKDataComponent.IsDirty) return;
+
+            pbComponent.WearableUrns.Clear();
+
+            foreach (URN urn in playerSDKDataComponent.WearableUrns) { pbComponent.WearableUrns.Add(urn); }
+
+            pbComponent.EmoteUrns.Clear();
+
+            foreach (URN urn in playerSDKDataComponent.EmoteUrns) { pbComponent.EmoteUrns.Add(urn); }
+
+            ecsToCRDTWriter.PutMessage<PBAvatarEquippedData, PBAvatarEquippedData>(static (dispatchedPBComponent, pbComponent) =>
+            {
+                dispatchedPBComponent.WearableUrns.Clear();
```

Comment:
Cleaning is already done when the component is rented: it's defined in `AsProtobufResult` and `AsProtobufComponent`

Reply:
True, removed those unneeded calls